### PR TITLE
source-datadog contribution from topefolorunso

### PR DESCRIPTION
**Pull request by topefolorunso**
## What

This PR updates source Datadog (source-datadog).

The contributor provided the following description of the change:

removed `is_read_only` attribute from dashboards stream schema
## Reviewer checklist
- [ ] Resolve any merge conflicts and validate file diffs (make sure the PR only includes changes intended by the contributor)
- [ ] After reviewing the changes, run the [`bump-version` Airbyte-CI command](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md#connectors-bump-version-command) locally to update the version of the connector according to the [versioning guidelines](https://docs.airbyte.io/contributor-guide/versioning-guidelines). Add `breakingChanges` to metadata if necessary.
- [ ] Ensure connector docs are up to date with any changes
- [ ] Run `/format-fix` to resolve any formatting errors
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

File: airbyte-integrations/connectors/source-datadog/manifest.yaml [hunk 1]
```diff
@@ -1,7 +1,9 @@
-version: 5.7.5
+version: 6.7.2
```
Comment (ChristoGrab) [inline airbyte-integrations/connectors/source-datadog/manifest.yaml:1]:
```suggestion
version: 5.16.0
```

**Comment by ChristoGrab** [comment]
@natikgadzhi, @DanyloGL and I had a chance to review, everything seems good. Required QA checks are failing due to the upgrade deadline being too close, but that's an API restriction (not on our end) so we just need to get an /approve-and-merge.

**Comment by natikgadzhi** [comment]
Done!

**Pull request merged**